app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "NotVaccinatedError"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "OutdatedVaccineError"
```

Comment:
The `__str__` method for `OutdatedVaccineError` should return a more descriptive message. You might want to return 'Visitor's vaccine is outdated.' to give a clear indication of the issue.